Apps/W1/HybridGP/app/Permissions/HybridGPEdit.PermissionSet.al

```diff
@@ -67,9 +67,11 @@ permissionset 4031 "HybridGP - Edit"
                     tabledata "GP Migration Errors" = IMD,
                     tabledata "GP Segment Name" = IMD,
                     tabledata "GP Company Additional Settings" = IMD,
-                    tabledata "GP SY40100" = RIMD,
-                    tabledata "GP SY40101" = RIMD,
-                    tabledata "GP CM20600" = RIMD,
+                    tabledata "GP SY40100" = IMD,
```

Comment:
This could be a breaking change if we have shipped with it.  If it is needed and you feel comfortable that nobody could have taken dependency on it we can change the permissions

Reply:
This corrects a mistake from a previous commit. All of the permissions here are IMD, and this permissionset includes 'HybridGP - Read', so having 'R' in this file is redundant.